sci-libs/calculix-cgx/files/01_cgx_2.10_Makefile_custom_cxx_flags.patch

```diff
@@ -1,30 +1,30 @@
-diff -rupN CalculiX/cgx_2.7/src/Makefile CalculiX.patched/cgx_2.7/src/Makefile
---- CalculiX/cgx_2.7/src/Makefile	2014-02-18 13:47:05.000000000 -0500
-+++ CalculiX.patched/cgx_2.7/src/Makefile	2014-09-04 17:08:58.043266103 -0400
+diff -uwrN cgx_2.10.ORIG/src/Makefile cgx_2.10/src/Makefile
+--- src/Makefile	2016-01-13 12:41:40.398930000 +0100
++++ cgx_2.10/src/Makefile	2016-03-08 11:02:59.591853482 +0100
 @@ -1,15 +1,15 @@
 -CFLAGS = -O3 -Wall \
-+CFLAGS := -O3 -Wall \
++CFLAGS += -O3 -Wall \
```

Comment:
I've just used what the author of the lib have done thinking "there might be a reason".